Cartridges/int_checkoutcom_sfra/cartridge/scripts/helpers/eventsHelper.js

```diff
@@ -77,6 +81,12 @@ var eventsHelper = {
                     order.setPaymentStatus(order[paymentStatus]);
                 }
 
+                // Update order if flagged
+                if (flagDetails) {
+                    order.addNote(ckoHelper._('cko.webhook.info', 'cko'), flagDetails);
+                    order.setConfirmationStatus(0);
```

Comment:
What does this number mean ? Could you use a constant which indicate some meaningful words